Sources/SwiftSyntaxMacrosTestSupport/Assertions.swift

```diff
@@ -322,10 +322,15 @@ public func assertMacroExpansion(
 
   // Expand all macros in the source.
   let context = BasicMacroExpansionContext(
+    lexicalContext: /*FIXME:*/ [],
```

Comment:
How should we compute the lexical context here? It seems like you are using the lexical context in your test cases.

Reply:
The context generator will create a fresh context instance with the appropriate lexical context. I'm going to default this parameter so we don't need this here.